kafka-connect-azure/pom.xml

```diff
@@ -0,0 +1,264 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>kafka-connect-storage-cloud</artifactId>
+        <version>4.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kafka-connect-azure</artifactId>
+    <packaging>jar</packaging>
+    <name>kafka-connect-azure</name>
+
+    <description>
+        Kafka Connect cloud storage connector for Azure blob storage.
+    </description>
+
+    <properties>
+        <azure-storage.version>6.1.0</azure-storage.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>findify</id>
+            <url>https://dl.bintray.com/findify/maven/</url>
+        </repository>
+    </repositories>
+
+    <dependencyManagement>
+        <dependencies>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-connect-s3</artifactId>
```

Comment:
Is this dependency kept until there is a `common-cloud` dependency?

Reply:
Yes, this was suggested by @kkonstantine here: https://github.com/confluentinc/kafka-connect-storage-cloud/issues/124#issuecomment-344676197